e2e/e2e_test.go

```diff
@@ -568,67 +565,6 @@ func testE2E() {
 		Expect(pod.Spec.NodeName).To(Equal(""))
 	})
 
-	It("should mount inline ephemeral volumes backed by LVMs to the pod and delete LVMs when pod is deleted", func() {
```

Comment:
Though you just deleted the test of inline ephemeral volume's test, don't you need to replace it with a test of generic ephemeral volume?

Reply:
In the case of Generic Ephemeral Volume, the ephemeral Controller simply creates the PVC corresponding to the Generic Ephemeral Volume.
https://github.com/kubernetes/kubernetes/blob/v1.24.1/pkg/controller/volume/ephemeral/controller.go#L241-L289

Therefore, Generic Ephemeral Volume is created/deleted (or reclaimed) based on the specified Storage Class setting like normal PVC, and there is no difference from normal PVC.

so, it seems that there is no need to add a special test case for Generic Ephemeral Volume on the CSI driver side.